src/autoscaling/snapshotter.js

```diff
@@ -312,7 +318,8 @@ class Snapshotter {
             const now = new Date();
             this._pruneSnapshots(this.clientSnapshots, now);
 
-            const currentErrCount = apifyClient.stats.rateLimitErrors;
+            const allErrorCounts = apifyClient.stats.rateLimitErrors;
+            const currentErrCount = allErrorCounts[2]; // 2nd retry.
```

Comment:
@mnmkng Constants must never be written directly in the code, always use a CONSTANT